### PR TITLE
populate-owners: resolve owner aliases

### PR DESCRIPTION
**Pull request by pgier**
This changes the populate-owners tool to resolve OWNER_ALIASES into the individual OWNERS files.  This avoids some complexity related to merging OWNER_ALIASES from different repositories, and also makes it easy to update OWNERS only for certain repositories.

### CODE REVIEW

**Comment by pgier** [comment]
CC @wking

**Comment by pgier** [comment]
See also #1601 and #1285

File: tools/populate-owners/main_test.go [hunk 1]
```diff
@@ -258,179 +262,27 @@ func TestExtractOwners(t *testing.T) {
...
-			} else if !test.error.MatchString(err.Error()) {
-				t.Fatalf("unexpected error: %v does not match %v", err, test.error)
-			}
-
-			assertEqual(t, collected, test.collected)
-			if test.namespaced != nil {
-				assertEqual(t, test.input, test.namespaced)
-			}
-		})
+func TestResolveAliases(t *testing.T) {
```
Comment (stevekuznetsov) [inline tools/populate-owners/main_test.go]: Would love a test for the slice insert

**Comment by pgier** [comment]
@stevekuznetsov Does this look ok?  I added tests for the slice insert and also updated the readme.

**Comment by stevekuznetsov** [comment]
@wking any thoughts here?

**Comment by stevekuznetsov** [comment]
Sounds good.

/lgtm

**Pull request merged**